packages/runtime/runtime-utils/src/test/summarizerNodeWithGc.spec.ts

```diff
@@ -181,133 +181,4 @@ describe("SummarizerNodeWithGC Tests", () => {
             assert.deepStrictEqual(gcData, internalGCData, "GC data should be generated by calling getInternalGCData");
         });
     });
-
-    describe("summarize API", () => {
```

Comment:
Are test cases still valid (in modified form to reflect changes in behavior)?

Reply:
`summarize` by itself should not return any GC data anymore. I can run `getGCData` before `summarize` to test that summarize returns the same GC data as getGCData.